code/modules/hydroponics/fermenting_barrel.dm

```diff
@@ -65,6 +65,30 @@
 		to_chat(user, "<span class='notice'>You close [src], letting you draw from its tap.</span>")
 	update_icon()
 
+/obj/structure/fermenting_barrel/crowbar_act(mob/living/user, obj/item/I)
+	. = TRUE
+	if(!I.use_tool(src, user, 0))
+		return
+	TOOL_ATTEMPT_DISMANTLE_MESSAGE
+	if(I.use_tool(src, user, 50, volume = I.tool_volume))
+		TOOL_DISMANTLE_SUCCESS_MESSAGE
+		new /obj/item/stack/sheet/wood(loc, 30)
+		qdel(src)
+
+/obj/structure/fermenting_barrel/wrench_act(mob/living/user, obj/item/I)
+	. = TRUE
+	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
+		return
+	if(anchored)
+		WRENCH_UNANCHOR_MESSAGE
+		anchored = FALSE
+	else
+		if(!isfloorturf(loc))
+			user.visible_message("<span class='warning'>A floor must be present to secure [src]!</span>")
```

Comment:
A `to_chat` makes more sense here

Reply:
I just copied it from `dresser.dm`, should I change that one too?

Reply:
Would be cleaner. Now it will show a message to everybody that can see the user. Bit odd when it is a feedback message